### ISSUE TITLE
Is WAT still supported?

### ISSUE THREAD

**Issue by shatu** [nopper/wikibench#1]
Hello,

I've been looking for WAT's source code for quite a while. Is it still available? The current link issues a 404.

Thanks,
Shashank

**Comment by nopper** [comment]
Yes it's supported. I still didn't find any free time to polish its source code before publication. At the moment it's a private repo that's why you get 404.

**Comment by shatu** [comment]
Alright. Do you guys still provide access to WAT through an API? 
If yes, what's the procedure to procure access to your service for benchmarking purposes?

EDIT: Sorry, as I can see, you guys provide some sort of service here -> http://wikisense.mkapp.it/tag/disambiguate

Is there any online documentation about the API?

Thanks,
Shashank

**Comment by kszucs** [comment]
I'm interested in the source code too. Will it be available in the near future?

Thanks, 
    Krisztian
